nms/app/views/domain-proxy/CbsdEdit.js

```diff
@@ -0,0 +1,425 @@
+                        })
+                      }}
+                      input={<OutlinedInput />}
+                    >
+                      <MenuItem value={5}>5</MenuItem>
+                      <MenuItem value={10}>10</MenuItem>
+                      <MenuItem value={15}>15</MenuItem>
+                      <MenuItem value={20}>20</MenuItem>
+                    </Select>
+                  </FormControl>
+                </Grid>
+              </Grid>
+
+              <Grid item xs={6} container spacing={1}>
+                <Grid item xs={6}>
+                  <AltFormFieldSubheading label={'Frequencies'} />
+                </Grid>
+
+                <Grid item xs={6} className={classes.addFrequencyIconContainer}>
+                  <IconButton
+                    className={classes.frequencyPreferencesIcon}
+                    color="primary"
+                    onClick={() =>
+                      setCbsdFormData({
+                        ...cbsdFormData,
+                        frequenciesMhz: [
+                          ...cbsdFormData.frequenciesMhz,
+                          0,
+                        ],
+                      })
+                    }
+                  >
+                    <AddIcon />
+                  </IconButton>
+                </Grid>
+
+                {cbsdFormData.frequenciesMhz.map((value, index) => {
+                  return (
+                    <>
+                      <Grid item xs={11}>
+                        <OutlinedInput
+                          fullWidth
+                          type="number"
+                          disabled={isLoading}
+                          value={value}
+                          onChange={({ target }) => {
+                            const newValue = cbsdFormData.frequenciesMhz.map((item, i) => {
+                              return i !== index
+                                ? item
+                                : target.value;
+                            });
+                            setCbsdFormData({
+                              ...cbsdFormData,
+                              frequenciesMhz: newValue,
+                            });
+                          }}
+                        />
+                      </Grid>
+                      <Grid item xs={1}>
+                        {cbsdFormData.frequenciesMhz.length > 1 &&
+                          <IconButton
+                            className={classes.frequencyPreferencesIcon}
+                            color="primary"
+                            onClick={() => {
+                              const newValue = cbsdFormData.frequenciesMhz.filter((_, i) => i !== index);
+                              setCbsdFormData({
+                                ...cbsdFormData,
+                                frequenciesMhz: newValue,
+                              });
+                            }}
+                          >
+                            <DeleteIcon />
```

Comment:
Would be nice if the delete icon would be vertically centred
![Screenshot 2022-05-10 at 16 01 46](https://user-images.githubusercontent.com/102796295/167654202-afe9b80c-5c2f-4651-b53f-dbc701983217.png)

Reply:
Fixed